models/anomalies/re_data_alerts.sql

```diff
@@ -0,0 +1,17 @@
+select
+    'anomaly' as type,
+    {{ re_data.clean_blacklist('table_name', ['"'], '') }} as model,
```

Comment:
I think here BigQuery will have ` ticks instead of "

Reply:
yeah, thanks for this. will add it to the list of characters.